src/material-experimental/mdc-table/BUILD.bazel

```diff
@@ -65,17 +65,23 @@ ng_test_library(
     deps = [
         ":mdc-table",
         "//src/cdk/table",
-        "//src/material/paginator",
+        "//src/material-experimental/mdc-paginator",
         "//src/material/sort",
-        "//src/material/table",
         "@npm//@angular/platform-browser",
         "@npm//rxjs",
     ],
 )
 
 ng_web_test_suite(
     name = "unit_tests",
-    static_files = ["@npm//:node_modules/@material/data-table/dist/mdc.dataTable.js"],
+    static_files = [
```

Comment:
why do the new tests need these?

Reply:
Because the tests now use the MDC-based paginator which has an internal form field and button. These are the MDC js files that they depend on.